### PR TITLE
Compute sha 256 hash

### PR DESCRIPTION
**Pull request by koln67**
Compute the sha 256 hash in the concourse image build pipeline and pass it in as part of the JSON.

### CODE REVIEW

**Comment by koln67** [comment]
/hold

do not merge until there is enough time in the day to manually test the concourse task

File: concourse/pipelines/linux-image-build.jsonnet [hunk 1]
```diff
@@ -262,6 +262,15 @@ local imgpublishjob = {
             load_var: 'publish-version',
             file: 'publish-version/version',
           },
+	  {
+	    task: 'generate-hash',
+	    file: 'guest-test-infra/concourse/tasks/generate-hash.yaml',
```
Comment (ChaitanyaKulkarni28) [inline concourse/pipelines/linux-image-build.jsonnet:267]: we need to compute hash for windows images as well

Comment (koln67) [reply concourse/pipelines/linux-image-build.jsonnet:267]: done

**Pull request merged**